README.md

```diff
@@ -1 +1,64 @@
-# forta-challenge
\ No newline at end of file
+# Nethermind Forta Bot
+
+## Description
+
+This project implements a Forta bot to monitor bot deployments, updates, and deletions by Nethermind. The bot generates alerts whenever a bot is created, updated, or deleted, helping to ensure transparency and traceability of bot management activities.
+
+## Supported Chains
+
+- Ethereum (Mainnet and Testnets)
+- Any EVM-compatible chain
+
+## Alerts
+
+The bot generates the following alerts:
+
+1. **Bot Deployment**
+   - **Alert ID**: NETH-1
+   - **Description**: A new bot has been deployed by Nethermind.
+   - **Severity**: Info
+   - **Type**: Info
+   - **Metadata**:
+     - `agentId`: The ID of the deployed agent.
+     - `metadata`: Metadata associated with the agent.
+     - `chainIds`: Chain IDs the agent is deployed on.
+
+2. **Bot Update**
+   - **Alert ID**: NETH-2
+   - **Description**: An existing bot has been updated by Nethermind.
+   - **Severity**: Info
+   - **Type**: Info
+   - **Metadata**:
+     - `agentId`: The ID of the updated agent.
+     - `metadata`: Updated metadata associated with the agent.
+     - `chainIds`: Chain IDs the agent is deployed on.
+
+3. **Bot Deletion**
+   - **Alert ID**: NETH-3
+   - **Description**: A bot has been deleted by Nethermind.
+   - **Severity**: Info
+   - **Type**: Info
+   - **Metadata**:
+     - `agentId`: The ID of the deleted agent.
+
+## Test Data
```

Comment:
The Test Data section should contain example txs on which the bot would trigger. You can check here for an example
[Test Data](https://github.com/NethermindEth/Forta-Agents/tree/main/reentrancy-counter#test-data)